docs/models/nuclear/hydrogen_tank.rst

```diff
@@ -1,37 +1,52 @@
 Hydrogen Tank
 ================
 
-The DISPATCHES Hydrogen Tank Model
+The DISPATCHES Hydrogen Tank Model represents a steady state implementation of a compressed hydrogen gas tank. This tank model supports tank filling and emptying operations for a fixed duration assuming a constant flow.
```

Comment:
"steady-state" seems like a misnomer here. It is not a true steady-state model, but I'm not sure what the correct name would be. Maybe quasi-steady-state?